packages/SwingSet/src/kernel/vatManager/externalStore.js

```diff
@@ -0,0 +1,56 @@
+// @ts-check
+
+import {
+  makeStore,
+  makeWeakStore,
+  makeClosureExternalStoreMaker,
+} from '@agoric/store';
+
+// TODO: Give this parameters so that it can interact with the vat's liveslots
```

Comment:
There is so much TODO in this file that I don't understand what the purpose of merging this PR into master would be. Is this intended to grow into the real external one itself, or is this a lightweight in-memory analog of what the external one should do. The latter seems more plausible to me.